<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movieLibrary</class>
 <widget class="QWidget" name="movieLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>movieLibrary</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#movieLibrary {
background-color: rgb(114, 159, 207);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);
border: 0 solid rgb(85, 87, 83);
border-radius: 5px;</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frmMovieInfo">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#frmMovieInfo {
	background-color: rgb(46, 52, 54);
	border: 0px solid;
	border-radius: 5px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblThumbnail">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
color: white;
qproperty-alignment: AlignTop;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblMovieTitle">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
color: white;
qproperty-alignment: AlignTop;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lblSynopsis">
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color: white;
qproperty-alignment: AlignTop;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblCast">
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color: white;
qproperty-alignment: AlignTop;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frmWhatisthis">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#frmWhatisthis {
	background-color: rgb(238, 238, 236);
	border: 0px solid;
	border-radius: 5px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="lblWhatisthis">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
qproperty-alignment: AlignTop;</string>
            </property>
            <property name="text">
             <string>What movie is this?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbWhatMovieIsThis"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frmTwitter">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#frmTwitter {
	background-color: rgb(238, 238, 236);
	border: 0px solid;
	border-radius: 5px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lblTwitterFeed">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Twitter Feed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView">
            <property name="styleSheet">
             <string notr="true">border: 0px; background-color:transparent</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">/*color: rgb(238, 238, 236);*/
color:white</string>
       </property>
       <property name="text">
        <string>Welcome to the Movie Library</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="status_one">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_two">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_three">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_four">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_five">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_six">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_seven">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_eight">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_nine">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_ten">
       <property name="styleSheet">
        <string notr="true">color:white;
qproperty-alignment: AlignCenter;
padding-left: 15px;
padding-right: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/configure-3.png</normaloff>:/configure-3.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
